fuzzing/Makefile.fuzzing_common

```diff
@@ -0,0 +1,19 @@
+RIOTBASE ?= $(CURDIR)/../..
+
+# Automatically set application to a sensible default
+APPLICATION ?= fuzzing_$(notdir $(patsubst %/,%,$(CURDIR)))
+
+# Fuzzing is only supported on native
+override BOARD = native
```

Comment:
You probably want to use `FEATURES_REQUIRED += arch_native` here.

Reply:
Fixed, thanks.

Reply:
This was caused by this line. Use
```suggestion
BOARD ?= native
```

`FEATURES_REQUIRED` below ensures the user will get a warning that their board is not required and the CI is able to build this ;-).

Reply:
Ahhhhhh, pushed thanks!